app/editor/src/features/admin/filters/FilterFormQuery.tsx

```diff
@@ -163,6 +168,41 @@ export const FilterFormQuery: React.FC = () => {
             </p>
           </Row>
         </Row>
+        <Row>
+          <Col flex="1">
+            <FormikText
+              name="edition"
+              label="Edition"
+              value={values.settings.edition ?? ''}
+              onChange={(e) => {
+                const value = e.target.value.length ? e.target.value : undefined;
```

Comment:
Empty string values are set as undefined so that they are not included in the query.